packages/backend/src/tools/updateDiffHistory.ts

```diff
@@ -36,14 +37,42 @@ async function updateDiffHistoryFile() {
     getFileVersionOnMainBranch(`${discoveryFolder}/discovered.json`)
   const discoveryFromMainBranch =
     DiscoveryJsonFromMainBranch === ''
-      ? { contracts: [] }
+      ? undefined
       : (JSON.parse(DiscoveryJsonFromMainBranch) as DiscoveryOutput)
   const diff = diffDiscovery(
-    discoveryFromMainBranch.contracts,
+    discoveryFromMainBranch?.contracts ?? [],
     curDiscovery.contracts,
     config,
   )
 
+  // To check for changes to source code,
+  // download sources for block number from main branch
+  let codeDiff = ''
```

Comment:
This doesn't need to be defined here. Also it does not need to be a `let`, can easily be `const`.